client/src/shared/components/Table/ListItem.tsx

```diff
@@ -31,10 +32,14 @@ export const ListItem: any = <T extends Data>({
   isChecked = false,
   checkboxName = '',
   onCheckboxChange,
+  isClickable,
   register,
 }: ListItemProps<T>) => {
   return (
-    <tr className="even:bg-neutral-50 whitespace-nowrap text-sm text-black1 font-sans h-5 group transition-all ease-out duration-200">
+    <tr
+      className="even:bg-neutral-50 whitespace-nowrap text-[13px] text-lightGray3 font-sans h-[49px] group transition-all ease-out duration-200 cursor-pointer"
+      onClick={isClickable}
+    >
```

Comment:
This will show `cursor-pointer` on all table rows including those that aren't clickable. You can refer to my previous comment: https://github.com/framgia/sph-lms/pull/133#discussion_r1230384928